src/main/java/io/mvnpm/esbuild/model/AutoEntryPoint.java

```diff
@@ -1,18 +1,16 @@
 package io.mvnpm.esbuild.model;
 
-import io.mvnpm.esbuild.util.QuteTemplateRenderer;
-
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.StandardOpenOption;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
 
 import static io.mvnpm.esbuild.util.Copy.copyEntries;
+import java.io.StringWriter;
+import java.io.UncheckedIOException;
 
 public class AutoEntryPoint implements EntryPoint {
     private static final Set<String> SCRIPTS = Set.of("js", "ts", "jsx", "tsx");
```

Comment:
@phillip-kruger 
We need to add those before releasing: (not related to your PR)
```java
Set.of("js", "ts", "jsx", "tsx", "mjs", "mts", "cjs", "cts");
```

Reply:
OK do you want to do a PR ?